CONTRIBUTING.md

```diff
@@ -1,178 +1,32 @@
-[string.seq](R-package/tests/testthat/testdata/string.seq), and then use any [transformation functions](https://tensorflow.rstudio.com/tools/tfdatasets/articles/introduction.html#transformations) provided by [tfdatasets package](https://tensorflow.rstudio.com/tools/tfdatasets/) on the dataset like the following:
-
-```r
-library(tfio)
-dataset <- sequence_file_dataset("R-package/tests/testthat/testdata/string.seq") %>%
-    dataset_repeat(2)
-
-sess <- tf$Session()
-iterator <- make_iterator_one_shot(dataset)
-next_batch <- iterator_get_next(iterator)
-
-until_out_of_range({
-  batch <- sess$run(next_batch)
-  print(batch)
-})
-```
-
-## Coding guidelines and standards
-
-### Naming Conventions
-
-Any new operation that is introduced in Tensorflow I/O project should be prefixed with a pre-defined text.
-This is done to ensure that different Tensorflow projects (for e.g Tensorflow I/O) declare their operation names that are globally unique,
-across the entire Tensorflow ecosystem. 
-
-For more details on defining custom operations, please refer this [tensorflow RFC](https://github.com/tensorflow/community/pull/126).
 
-Depending on the module's programming language (C++ or python), pre-defined prefix text may vary. This is detailed below.
- 
-#### C++
-
-Any new operation that is introduced in C++ module should be prefixed with text “IO>”. 
-
-for example:
-
-A new operation named “ReadAvro” should be registered as “IO>ReadAvro”.
-
-##### Regular way:
-
-```text
-REGISTER_OP("ReadAvro")
-                    ...
-                    ...
-                    ...
-     });
-```
-
-##### Suggested way:
-
-```text
-REGISTER_OP("IO>ReadAvro")
-                    ...
-                    ...
-                    ...
-     });
-```
-
-Please note in suggested way, how the operation name is prefixed with "IO>".
-
-#### Python 
+## Contributor License Agreement
 
-Any new operation that is introduced in python module and that which corresponds to the operation in C++ module,
-should be prefixed with "io_". This ensures that C++ operation binding is mapped correctly.
+Contributions to this project must be accompanied by a Contributor License
+Agreement. You (or your employer) retain the copyright to your contribution;
+this simply gives us permission to use and redistribute your contributions as
+part of the project. Head over to https://cla.developers.google.com/ to see
+your current agreements on file or to sign a new one.
 
-for example:
+You generally only need to submit a CLA once, so if you've already submitted one
+(even if it was for a different project), you probably don't need to do it
+again.
 
-The python equivalent declaration of above operation named “IO>ReadAvro” would be "io_read_avro".
+## Coding Style
 
-```text
-def read_avro(filename, schema, column, **kwargs):
-  """read_avro"""
-  ...
-  ...
-  ...
-  return core_ops.io_read_avro(...);
-```
+Please see our [Style Guide](STYLE_GUIDE.md) for more details.
```

Comment:
We have a project-wide code style guide here: https://www.tensorflow.org/community/contribute/code_style
If these differ, can you reconcile and link here as well?

Reply:
Thanks @lamberta. Please let me know, if following text is what you are suggesting?

"Tensorflow project wide code style guidelines can be followed at [TensorFlow Style Guide - Conventions](https://www.tensorflow.org/community/contribute/code_style) 
and Tensorflow I/O project specific code style guidelines can be followed at [Style Guide](STYLE_GUIDE.md)."

Reply:
That works—thanks!

Reply:
Updated as suggested! Thanks.